models/validators/InListValidator.cfc

```diff
@@ -23,13 +23,25 @@ component accessors="true" implements="cbvalidation.models.validators.IValidator
 	* @validationDataThe validation data the validator was created with
 	*/
 	boolean function validate(required cbvalidation.models.result.IValidationResult validationResult, required any target, required string field, any targetValue, any validationData){
+		if ( isNull( arguments.targetValue ) ) {
```

Comment:
But isn't this a false? The actual target value is NOT in the list because it is null. So I am not sure that it should be a true.

Reply:
If you need to make sure a value isn’t null, add a `required` rule. I need this to be able to be used with optional params.

Reply:
that makes sense, thanks for the clarification